tools/state-manager/test_restore_runner.py

```
@@ -0,0 +1,112 @@
+    python3 -m unittest test_restore_runner -v
+"""
+
+import sys
+import tempfile
+import unittest
+from pathlib import Path
+from types import SimpleNamespace
+from unittest import mock
+
+
+def _make_k8s_stub():
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_make_k8s_stub`

(ANN202)